thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/tools/RunAdhocDatabaseQueriesTool.java

```diff
@@ -225,6 +233,12 @@ private void updateDetectionRun(String dataset) {
     }
   }
 
+  private void updateAlertFilterForFunctionId(long functionId, Map<String, String> alertFilter) {
```

Comment:
Are these functions in this file is only for local test purpose? if not move to under main/